docs/Targets.md

```diff
@@ -1053,6 +1053,31 @@ DCP support (hardware acceleration for SHA256 operations) can be enabled by usin
 
 Firmware can be directly uploaded to the target by copying `factory.bin` to the virtual USB drive associated to the device, or by loading the image directly into flash using a JTAG/SWD debugger.
 
```

Comment:
We should probably add some documentation here about which of the new flash configuration macros to set  (`USE_GET_CONFIG`, `CONFIG_FLASH_IS25WP064A`, `CONFIG_FLASH_W25Q64JV`, `CONFIG_FLASH_QE_ENABLE`), as well as clarify whether or not we support HyperFlash.